pkg/migration/controllers/migration.go

```diff
@@ -2735,3 +2738,79 @@ func isObjectHashEqual(dynamicClient dynamic.ResourceInterface, u *unstructured.
 	}
 	return false
 }
+
+// getRelatedCRDListWRTGroupAndComponents takes input of AppRegs
+// and find out all CRDs those need to be collected.
+// The CRD list should contain
+//
```

Comment:
nit..Just rewording a bit
getRelatedCRDListWRTGroupAndCategories takes AppRegs as input and 
finds out all the CRDs that need to be collected.
This CRD list should contain
* Also remove the space after this line

Reply:
Done